README.md

```diff
@@ -1 +1,56 @@
-# leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> This is the displaying scores submitted by different players. It also allows users to add their score. The main goal of this project was to be able to deal with API and play with promises and async/await. For best practices I have implemented Linter,GitFlow and professional documentation. 
+
+![screenshot](./images/Leaderboard.png)
+
+## Built With
+
+- HTML, CSS, js
+- Linters, Markdown
+- GitFlow 
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1) Enter `git@github.com:basnetlaxmi/leaderboard.git` into terminal
+
+2) Open in browser to view site
```

Comment:
- [ ] Please write proper steps to run the project locally. Please include the steps in the README.

```
$ git clone https://github.com/basnetlaxmi/leaderboard.git
$ cd leaderboard

$ npm i
$ npm start
```